champs_des_alpes.cup

```diff
@@ -10,6 +10,7 @@ name,code,country,lat,lon,elev,style,rwdir,rwlen,rwwidth,freq,desc,userdata,pics
 "Oex - Sallanches 2",SallaOex,FR,4558.682N,00637.943E,535.0m,5,0,0.0m,,,"Piste en travers de la vallee.Deux zones de 400m utilisables dans axe de la vallee.Attention cabane entre deux zones. Partie ouest piste ULM est montante et le sol est plan dans toute la plaine.",,
 "Richardiere",,FR,4448.774N,00534.275E,0.0m,3,0,0.0m,,,"La Richardiere",,"fugieres.jpg"
 "Envie Aero",Enviaero,IT,4441.733N,00724.150E,0.0m,2,0,0.0m,,,"Terrain prive.",,"envie.jpg"
+"Valenty2", valenty2,FR,4422.500N,00556.000E,568.0m,2,026,480m,,,"bonne alternative à Valenty",,"valenty2.jpg"
```

Comment:
Il faut que tu ajoutes le fichier valenty2.jpg dans le repertoire /Pics
Image au format 748x748 de preference de Geoportail (qui est libre de droits)
Pour coherence avec les autres commentaires:
nom = Valenty_2
Type = 3 (outlanding place) au lieu de 2 (airport with grass runway)
Commentaire = Majuscule sur la 1er lettre: Bonne alternative ...

Reply:
OK pour le jpeg, tu as fait les corrections mentionnées ou pas ?

Reply:
Non je n'ai pas fait les corrections. Je te laisse corriger, et refaire un commit.
En fait pour que cela soit plus simple il faut faire une branche par "changement" et ensuite tu fais une pull request sur ta branche.
Du coups si il y a plusieurs changement, cela reste la meme pull request (merge de ta branche vers le master).
Apres il faut que tu merges ta branch dans ton master a toi.

Reply:
J'ai un fichier corrigé dans mon répertoire, mais je ne sais pas comment faire pour générer une nouvelle pull request...?

Reply:
C'est bon, il y a les 2 autres commit dans cette pull request. Maintenant il faut que tu ajoutes le fichier jpg et ca devrait apparaitre dans la pull request...

Reply:
Le monsieur me dit : File uploads require push access to this repository. 
Quand je vais dans le répertoire pics.

Reply:
C'est dans ton repo a toi (celui qui est un fork du mien) que tu dois ajouter le fichier, et ensuite tu fais une pull request vers mon repo pour l'ajouter. Dans ton repo on dirait que c'est la branch patch-7 qui est celle des changements de Valenty

Reply:
Mais je n'arrive pas vraiment à aller das mon repo à moi..., ou alors par accident !

Reply:
Il est la: https://github.com/delta217/outlanding

Reply:
çà y est, j'ai pigé comment passer de l'un à l'autre...
Bac+7, c'est pas assez, plutôt bac+10...

Reply:
Je viens de regarder. Ce qui va bien:
Tu as fais des changements dans ton repo, ca c'est bien.
Ce qui va moins bien:
1) Tu as fait une pull request https://github.com/delta217/outlanding/pull/1 qui est un changement de ton repo vers ton repo. Ce n'est pas nécessaire si tu es seul a travailler dans ton repo.
2) Tu as ajoute des images directement dans ton repo: tres bien, mais il faut ensuite faire une pull request de ton repo vers le mien pour que je puisse en beneficier.,
3) Ton repo est maintenant bien "en retard" par rapport au mien (27 commits behind). Ca veut dire que ton repo n'est plus a jour de sa base. Il faut regulierement se "rebaser" sur le repo d'origine. Il faut donc que tu clicks sur "27 commits behind" et voir si tu peux faire un rebase.

En fait de maniere generale voila comment il faut travailler:
Tu as ton repo qui est "a jour" par rapport au mien
Tu fais une branche dans ton repo
Tu modifies ce que tu veux dans ta branche
Tu fais une pull request de ta branche vers le "main" de mon repo
Je regarde la pull request, je fais des commentaires
Tu fais des modification suite aux commentaires dans cette meme branche (la pull request reste donc la meme)
J'accepte la pull request et je merge = j'ai donc tes modifications
Tu merges ta branche dans ton "main" = tu as integre tes modifications dans ton main (branche principale)
et ainsi de suite...

Le plus simple est peut etre d'en parler par telephone si tu veux. Envois moi un mail, donnes moi ton et dis moi quand je peut te joindre. ludovic.launer@gmail.com

Merci pour ton travail et ta volonté d'apprendre comment marche github !

Reply:
1 et 2, je crois que c'est bon...
3, allô ! 06 71 93 60 75